stock_view_adj_hls/views/stock_quant_views.xml

```diff
@@ -25,4 +25,16 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_stock_move_line_operation_tree" model="ir.ui.view">
+        <field name="name">stock.move.line.operations.tree</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id"
+               ref="stock.view_stock_move_line_operation_tree"/>
+        <field name="arch" type="xml">
+            <field name="lot_id" position="attributes">
+                <attribute name="readonly">1</attribute>
```

Comment:
Any reason this should be readonly instead of invisible?

Reply:
@yostashiro 
I think there is a situation where want to check the lot_id, so I set it to readonly.